src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/ApiMapType.java

```diff
@@ -155,12 +155,13 @@ public ApiMapType create(MapType cqlType, VectorizeDefinition vectorizeDefn)
     public boolean isSupported(MapType cqlType) {
       Objects.requireNonNull(cqlType, "cqlType must not be null");
       // we accept frozen and then change the support
-      // keys must be text or ascii, because keys in JSON are string
-      if (!(cqlType.getKeyType() == DataTypes.TEXT || cqlType.getKeyType() == DataTypes.ASCII)) {
-        return false;
-      }
       // must be a primitive type value
       return cqlType.getValueType() instanceof PrimitiveType;
     }
   }
+
+  @Override
+  public boolean isFrozen() {
```

Comment:
code tidy : move this to be above the innner clases so it is with the other overrides from the same class . do this for all places